src/components/container/BloodPressureVisualization/BloodPressureVisualization.css

```diff
@@ -0,0 +1,79 @@
+body{
+    background: #FFFFFF;
+}
+
+.mdhui-blood-pressure-visualization-pager {
+    width: 100%;
+    padding-top: 13px;
+    padding-bottom: 13px;
+}
+
+.mdhui-blood-pressure-metrics {
+    padding-top: 10px;
+    padding-bottom: 10px;
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+}
+
+.mdhui-blood-pressure-metrics-rows{
+    display: flex;
+    flex-direction: row;
+    background-color: #FFFFFF;
```

Comment:
This shouldn't need a background color and if it did, would use the variable for this so it works in dark mode